aws/ec2/resources.py

```diff
@@ -117,7 +118,8 @@ def ec2_security_groups_with_in_use_flag():
 
     # These resources have two types of security groups, therefore
     # the Vpc ones are namespaced under "VpcSecurityGroups"
-    vpc_namespaced_resources = sum([rds_db_instances(), redshift_clusters()], [])
+    vpc_namespaced_resources = sum(
+        [rds_db_instances(), redshift_clusters()], [])
```

Comment:
this and the above are automated formatting from vscode. should I remove them?

Reply:
Yeah, we use [black](https://github.com/ambv/black) for formatting, with https://github.com/mozilla-services/pytest-services/blob/master/Makefile#L51-L52 checking in CI.